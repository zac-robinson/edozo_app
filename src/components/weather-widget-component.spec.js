import React from 'react';
import { shallow } from 'enzyme';

import WeatherWidget from './weather-widget-component';

describe('WeatherWidget component', () => {
  it('should render correctly', () => {
    const props = {
      city: {
        name: 'York'
      },
      temperature: {
        temp: 45
      },
      wind: {
        speed: 3444
      },
      conditions: {
        description: 'still raining'
      }
    };
    const component = shallow(<WeatherWidget {...props} />);

    expect(component).toMatchSnapshot();
  });
});
