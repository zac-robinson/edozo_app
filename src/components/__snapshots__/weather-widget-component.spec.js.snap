// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherWidget component should render correctly 1`] = `
<styled.div>
  <WeatherWidgetHeader
    city={
      Object {
        "name": "York",
      }
    }
    conditions={
      Object {
        "description": "still raining",
      }
    }
    temperature={
      Object {
        "temp": 45,
      }
    }
    wind={
      Object {
        "speed": 3444,
      }
    }
  />
</styled.div>
`;
